packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -413,6 +413,35 @@ export async function launch({
 
   // Not to be confused with the gas model, this meter is for OpenTelemetry.
   const metricMeter = metricsProvider.getMeter('ag-chain-cosmos');
+
+  // Define the action types and their corresponding metric names
+  const actionMetrics = {
+    CORE_EVAL: metricMeter.createCounter('action_core_eval_total', {
```

Comment:
Let's use the `ActionType` to avoid more hard coded strings

```suggestion
    [ActionType.CORE_EVAL]: metricMeter.createCounter('action_core_eval_total', {
```

Reply:
Yes, good point.

Reply:
done